internal/provider/apiproductversion_data_source.go

```diff
@@ -62,7 +62,7 @@ func (r *APIProductVersionDataSource) Schema(ctx context.Context, req datasource
 			"deprecated": schema.BoolAttribute{
 				Computed:           true,
 				DeprecationMessage: `This will be removed in a future release, please migrate away from it as soon as possible`,
-				Description:        `Whether this API product version is deprecated in at least one portal. This field is deprecated: Use [PortalProductVersion.deprecated](https://docs.konghq.com/konnect/api/portal-management/latest/#/Portal%20Product%20Versions/create-portal-product-version) instead`,
+				Description:        `Whether this API product version is deprecated in at least one portal. This field is deprecated: Use [PortalProductVersion.deprecated](https://docs.konghq.com/konnect/api/portal-management/v2/#/operations/create-portal-product-version) instead`,
```

Comment:
Seems like the recommended endpoint here is updated in description

Reply:
This is intentional